examples/demo-apps/android/LlamaDemo/app/src/main/java/com/example/executorchllamademo/LlmBenchmarkRunner.java

```diff
@@ -106,14 +106,14 @@ public void onGenerationStopped() {
         new BenchmarkMetric(
             benchmarkModel,
             "model_load_time(ns)",
```

Comment:
Nit: The metric value is now in (ms) instead of (ns), so its name need to be updated accordingly